src/app/app.component.html

```diff
@@ -1,3 +1,179 @@
-<h1>
-  {{title}}
-</h1>
+<!-- ********** start ********* -->
+<md-sidenav-container class="width100">
```

Comment:
filename is not correct. 

put this is app/components

app/modules/index app/modules/login